karapace/protobuf/proto_normalizations.py

```diff
@@ -77,28 +78,41 @@ class NormalizedGroupElement(GroupElement):
 class NormalizedProtobufSchema(ProtobufSchema):
     proto_file_element: NormalizedProtoFileElement
 
-    def __init__(
-        self,
-        schema: str,
-        references: Sequence[Reference] | None = None,
-        dependencies: Mapping[str, Dependency] | None = None,
-        proto_file_element: ProtoFileElement | None = None,
-    ) -> None:
-        super().__init__(schema, references, dependencies, proto_file_element)
-        self.proto_file_element = normalize(self.proto_file_element)
+    @staticmethod
+    def from_protobuf_schema(protobuf_schema: ProtobufSchema) -> NormalizedProtobufSchema:
+        return normalize(protobuf_schema)
 
 
 class NormalizedOneOfElement(OneOfElement):
     fields: Sequence[NormalizedFieldElement]
     groups: Sequence[NormalizedGroupElement]
 
 
-def type_field_element_with_sorted_options(type_field: FieldElement) -> NormalizedFieldElement:
+def type_field_element_with_sorted_options(
```

Comment:
Should we consider renaming this to `normalize_type_field_element` as it is not exclusively sorting options anymore?